hydra/_internal/core_plugins/bash_completion.py

```diff
@@ -16,7 +16,11 @@
 class BashCompletion(CompletionPlugin):
     # TODO: detect python with path like /foo/bar/python
     def install(self) -> None:
-        script = """hydra_bash_completion()
+        # Record the old rule for uninstalling
+        script = (
+            f"export _HYDRA_OLD_COMP=$(complete -p {self._get_exec()} 2> /dev/null)\n"
+        )
+        script += """hydra_bash_completion()
```

Comment:
since you are improving this, can you add a test as well?
you can verify state by grepping the output of complete.

Reply:
Can you suggest a change? I don't get it what to test?
The output is like
```
$ complete -p python
complete -o bashdefault -o default -F _fzf_path_completion python
```
or if the completion doesn't exist
```
$ complete -p notexist
bash: complete: notexist: no completion specification
```

Reply:
This is what I had in mind:

You preserve the old completion and restore it on uninstall.
test:
0. clear all completion
1. configure some completion for python (not Hydra).
2. install Hydra bash completion
3. uninstall Hydra bash completion
4. verify your the old completion is restored.